mono/metadata/mono-gc.h

```diff
@@ -92,6 +92,10 @@ typedef enum {
 	 * The \c key parameter is a thread ID as a \c uintptr_t.
 	 */
 	MONO_ROOT_SOURCE_HANDLE = 14,
+	/**
+	 * Roots in the ephemeron arrays. This is a pseudo-root.
+	 */
+	MONO_ROOT_SOURCE_EPHEMERON = 15,
```

Comment:
This needs to have a corresponding value here: https://github.com/mono/mono/blob/master/mcs/class/Mono.Profiler.Log/Mono.Profiler.Log/LogEnums.cs#L155